REST Bindings/openapi.yaml

```diff
@@ -176,6 +176,21 @@ paths:
                         enum:
                           - epcisException:ValidationException
                   - $ref: '#/components/schemas/RFC7807ProblemResponseBody'
+            application/problem+xml:
```

Comment:
why don't we reuse the existing schema?

Reply:
we are, it's just a copy for media type application/problem+xml: - $ref: '#/components/schemas/RFC7807ProblemResponseBody'